tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -38,12 +38,27 @@ struct spi_cs_control spi_cs = {
 
 #define STACK_SIZE 512
 #define BUF_SIZE 17
+#define BUF2_SIZE 36
+
+#if CONFIG_NOCACHE_MEMORY
+static __aligned(32) uint8_t buffer_tx[BUF_SIZE] __used
+	__attribute__((__section__(".nocache")));
+static const uint8_t BUFFER_TX[] = "0123456789abcdef\0";
+static __aligned(32) uint8_t buffer_rx[BUF_SIZE] __used
```

Comment:
what about define the const out of the #if so for both configs. and not initialize the buffer2_tx in the #else